script.js

```diff
@@ -71,10 +60,43 @@ function makePageForEpisodes(episodeList) {
     rootElem.appendChild(footer)
     footer.appendChild(link)
   }
+  let showSelect=document.querySelector("#selectedshow")
+  allShows.forEach((show) => {
+    const option = document.createElement("option"); // create option element for each ep and fill the select dropdown
+    showSelect.appendChild(option);
+    option.innerText = `${show.name}`
+    option.setAttribute('value',show.id)
+  })
+   showSelect.addEventListener('change',()=>{
+    allShows.forEach(show=>{
```

Comment:
you didnt use "show" in the foreach loop and also the url. if you change them what will happend?